src/app/components/settings-modal.component.html

```diff
@@ -8,6 +8,14 @@ <h4 class="modal-title">Settings</h4>
       <label class="custom-control-label" for="analytics">I agree to share anonymous usage data</label>
     </div>
     <p class="text-muted"><small>We are using a basic Google Analytics setup, none of your data/content is kept anywhere and it helps us improve Mockoon :)</small></p>
+
+    <div>
+      <label for="log-body-size">Log body size:</label>
```

Comment:
Could you add a small explanation (with the unit, bytes?) after the label? Like this:
`<p class="text-muted"><small>Truncate the entering requests body if bigger (bytes).</small></p>`

Reply:
I add "Truncate the entering requests and exiting responses bodies if bigger (characters)." because it's limit the size of response body log too. What you think?

Reply:
This is perfect, even clearer :)